### DISCUSSION TITLE
Skupper go sdk

### DISCUSSION THREAD

**Discussion by HamzaZo** [skupperproject/skupper#796] [Q&A] [answered]
Hello,

I'm wondering if there is any go SDK like (k8s client-go ) to be able to use skupper ? Is there any example on how to use skupper using go ?

Thanks

**Comment by grs** [comment]
Can you elaborate a little on what you mean by 'use skupper using go'?

**Reply by HamzaZo** [reply]
Thanks for your reply.

Instead of using the CLI skupper manually to expose a service or to get the status, I would like to know if I can use skupper API to expose and get service status.

**Comment by grs** [comment]
You can annotate a deployment (or statefulset) and skupper will automatically expose it. We don't yet have a good way of programatically exposing the status.

**Reply by HamzaZo** [reply]
Can you share a doc that talks about using annotations?

**Reply by HamzaZo** [reply]
Can we also use annotations to link two namespaces ?

**Comment by grs** [comment, marked as answer]
https://skupper.io/docs/declarative/tutorial.html that also shows the process for linking two sites together
